examples/multimodel/hand_face_person_detection_parallel_video_stream.ipynb

```diff
@@ -0,0 +1,147 @@
+    "2. Run inference on DeGirum AI Server deployed on a localhost or on some computer in your LAN or VPN;\n",
+    "3. Run inference on DeGirum ORCA accelerator directly installed on your computer.\n",
+    "\n",
+    "To try different options, you need to specify the appropriate `hw_location` option. \n",
+    "\n",
+    "You also need to specify your cloud API access token in [env.ini](../../env.ini) file, located in the same directory as this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a1de047",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make sure degirum-tools package is installed\n",
+    "!pip show degirum-tools || pip install degirum-tools"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f04f85e-e516-4ab8-8ecc-33a6a86d85d2",
+   "metadata": {},
+   "source": [
+    "#### Specify where you want to run your inferences, model_zoo_url, model names for hand, face, and person detection, and video source"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "65d4cd90",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# hw_location: where you want to run inference\n",
+    "#     @cloud to use DeGirum cloud\n",
+    "#     @local to run on local machine\n",
+    "#     IP address for AI server inference  \n",
+    "# model_zoo_url: url/path for model zoo\n",
+    "#     cloud_zoo_url: valid for @cloud, @local, and ai server inference options\n",
+    "#     '': ai server serving models from local folder\n",
+    "#     path to json file: single model zoo in case of @local inference\n",
+    "# hand_det_model_name: name of the model for hand detection\n",
+    "# face_det_model_name: name of the model for face detection\n",
+    "# person_det_model_name: name of the model for person detection\n",
+    "# video_source: video source for inference\n",
+    "#     camera index for local web camera\n",
+    "#     URL of RTSP stream\n",
+    "#     URL of YouTube Video\n",
+    "#     path to video file (mp4 etc) \n",
+    "hw_location='@cloud'\n",
+    "model_zoo_url = 'https://cs.degirum.com/degirum/public'\n",
+    "hand_det_model_name= 'yolo_v5s_hand_det--512x512_quant_n2x_orca1_1'\n",
+    "face_det_model_name= 'yolo_v5s_face_det--512x512_quant_n2x_orca1_1'\n",
+    "person_det_model_name= 'yolo_v5s_person_det--512x512_quant_n2x_orca1_1'\n",
+    "video_source='../../images/person_face_hand.mp4'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "290deaa8",
+   "metadata": {},
+   "source": [
+    "#### The rest of the cells below should run without any modifications"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "33012fe3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Successfully opened video stream '../../images/person_face_hand.mp4'\n"
```

Comment:
Also it would be nice to click `Clear All Outputs` button on every notebook to avoid submitting some results of previous runs.

Reply:
Done